manifests/scenario/base.pp

```diff
@@ -114,7 +114,10 @@
     include '::postfix'
   }
 
-  if $ldap {
+  # Even if $ldap is true, if the host is on an IPA domain, do not include
+  # simp_openldap::client
+  # @see simp/simplib lib/facter/ipa.rb
+  if $ldap or !$facts['ipa'] {
```

Comment:
should this be 'if $ldap and !$facts['ipa'] {'

Reply:
I thought we decided to ignore the ldap catalyst?

Reply:
I see my error. Should be fixed now